mlrun/utils/notifications/notification/slack.py

```diff
@@ -36,7 +37,10 @@ class SlackNotification(NotificationBase):
     }
 
     @classmethod
-    def validate_params(cls, params):
+    def validate_params(cls, params, skip_empty_check=False):
+        if skip_empty_check:
+            mlrun.utils.logger.debug("Skipping emptiness check for notification params")
```

Comment:
I find this log line too verbose (spammy). Perhaps should be omitted completely (and for all notifications)

Reply:
I agree here, will remove 👍